web/src/beta/components/Blocks/ImageBlock/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { CSSProperties, FC, ImgHTMLAttributes } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+type Props = {
+  src: NonNullable<ImgHTMLAttributes<HTMLImageElement>["src"]>;
+  align?: "left" | "center" | "right"; // default center
```

Comment:
You shouldn't need the comments here in the props.
Can you 
- remove the comments
- Inside the props destructuring(line 15) assign defaults (ie. `({src, align="center", fit="cover", .. .. . .. })`
- Lastly update the styled component's props to expect a value for fit and align always and remove the fallbacks

Reply:
The code has changed.
Above code no longer exists.